doc/release/RELEASE-NOTES.md

```
@@ -3,6 +3,38 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html). The format is based on the `RELEASE-NOTES-TEMPLATE.md` file.
 
+## Release 1.24.0
+
+## Introduction
+* Product name: Open Supply Hub
+* Release date: November 16, 2024
+
+### Database changes
+#### Migrations:
```

Markdownlint

Punctuation: ':'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)